.github/workflows/checks.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Checks
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  checks:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 15.0.1
+        uses: actions/setup-java@v1
+        with:
+          java-version: 15.0.1
+          java-package: jdk+fx
+
```

Comment:
Should not be part of this PR

Reply:
as a note this was just added to allow for releasing via github actions for the test today and should disappear once the branch is rebased on develop